Remotion/Mixins/MixerTools.UnitTests/Mixer_IntegrationTest.cs

```diff
@@ -51,41 +51,37 @@ public void TearDown ()
     [Ignore("TODO RM-7799: Create out-of-process test infrastructure to replace tests done with app domains")]
```

Comment:
Might be disabled for a reason:

```
Remotion.Reflection.TypeDiscovery.AssemblyLoading.AssemblyLoaderException : The assembly 'Microsoft.VisualStudio.TestPlatform.ObjectModel, Version=11.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' (loaded in the context of 'NUnit3.TestAdapter, Version=4.0.0.0, Culture=neutral, PublicKeyToken=4cb40d35494691ac') triggered a FileNotFoundException - maybe the assembly does not exist or a referenced assembly is missing?
FileNotFoundException message: Could not load file or assembly 'Microsoft.VisualStudio.TestPlatform.ObjectModel, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.
```

Since the test won't run I cannot test if the assemblies are loaded. This should probably be a separate issue to resolve this error and remove the ignore. Since the mixer project is .NET Framework right now I would just revert the changes and once we convert the project to .NET we can rewrite the test